lib/scala/project-manager/src/main/resources/META-INF/native-image/org/enso/projectmanager/reflect-config.json

```diff
@@ -565,5 +565,51 @@
         "parameterTypes": ["java.lang.Boolean", "java.lang.Object"]
       }
     ]
+  },
+  {
+    "name": "org.enso.logging.LogbackLoggingServiceFactory",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
+    "name": "org.enso.logger.LogbackSetup",
```

Comment:
Alternative to specifying reflection here is to initialize `LoggerSetup.get` singleton in static initializer. Then it will be "wired" before native image is generated.